app/puppet-code/deploy.go

```diff
@@ -64,7 +64,7 @@ type DeployArgs struct {
 }
 
 func writeDeployResult(args *DeployArgs, payload []*operations.DeployOKBodyItems0) []byte {
-	fmt.Fprintf(os.Stderr, "Found %d environments.\n", len(payload))
+	fmt.Fprintf(os.Stdout, "Found %d environments.\n", len(payload))
```

Comment:
could we remove printing from sdk?

Reply:
It seems that [`WriteDeployResult`](https://github.com/puppetlabs/pe-sdk-go/blob/main/app/puppet-code/deploy.go#L66) function is called only in [`DeployWIthErrorDetails`](https://github.com/puppetlabs/pe-sdk-go/blob/main/app/puppet-code/deploy.go#L27) where the output is taken as it is and further returned. In the end, this is all printed to Stdout by puppet-code in [`deployCommand`](https://github.com/puppetlabs/pe-cli/blob/main/puppet-code/subcommands/deploy.go#L56) so I think that it should be safe to prepend the [json result payload](https://github.com/puppetlabs/pe-sdk-go/blob/main/app/puppet-code/deploy.go#L78) with [this information](https://github.com/puppetlabs/pe-sdk-go/blob/main/app/puppet-code/deploy.go#L67). Manually tested and it works as expected.